### ISSUE TITLE
TypeError: Got an unexpected keyword argument "maxItems"

### ISSUE THREAD

**Issue by christianbaumann** [jay0lee/got-your-back#177]
Full steps to reproduce the issue:
1.  Open cmd, go to directy where gyb.exe is located
2. Execute gyb --email YOURMAIL@gmail.com --action estimate
3. Secelct action 0, 5 and 6, press 7
4. Select & confirm Gmail account in browser

Expected outcome (what are you trying to do?): Size of Gmail is estimated

Actual outcome (what errors or bad behavior do you see instead?):
Using Windows 10
```
Authentication successful.
Traceback (most recent call last):
  File "gyb.py", line 1583, in <module>
  File "gyb.py", line 1537, in main
  File "gyb.py", line 532, in callGAPIpages
  File "gyb.py", line 485, in callGAPI
  File "googleapiclient\discovery.py", line 718, in method
TypeError: Got an unexpected keyword argument "maxItems"
[13072] Failed to execute script gyb
```

Using Ubuntu 17.10
```
Authentication successful.
Traceback (most recent call last):
  File "gyb.py", line 1583, in <module>
    main(sys.argv[1:])
  File "gyb.py", line 1537, in main
    maxItems=500, fields='nextPageToken,messages/id')
  File "gyb.py", line 532, in callGAPIpages
    pageToken=pageToken, **kwargs)
  File "gyb.py", line 485, in callGAPI
    method = getattr(service, function)(**parameters)
  File "/home/osboxes/Downloads/got-your-back-1.01/googleapiclient/discovery.py", line 718, in method
    raise TypeError('Got an unexpected keyword argument "%s"' % name)
TypeError: Got an unexpected keyword argument "maxItems"
```

**Comment by davidwhutton** [comment]
I have exactly the same problem -- on Windows 10.

**Comment by Radnet** [comment]
Me too, the same problem on Windows 10.

```
Traceback (most recent call last):
  File "gyb.py", line 1583, in <module>
  File "gyb.py", line 1537, in main
  File "gyb.py", line 532, in callGAPIpages
  File "gyb.py", line 485, in callGAPI
  File "googleapiclient\discovery.py", line 718, in method
TypeError: Got an unexpected keyword argument "maxItems"
[19444] Failed to execute script gyb
```

**Comment by christianbaumann** [comment]
According to this thread on google groups, the issue has been fixed: https://groups.google.com/forum/#!topic/got-your-back/-HlR7klfQkg

I would guess, a new release for windows is needed.

**Comment by Radnet** [comment]
Yes, that's correct, the issue has been fixed on the code but a new Windows executable wasn't generated.
I've made my own ,exe file using the latest version of the master branch and it works perfectly.

**Comment by jorhett** [comment]
He fixed it in commit 374a07539a1774060d8e1f9fc09c4bcf49fe69b9. It's a one-word change, easy to apply yourself.

**Comment by MTAydogdu** [comment]
Not if one has never done it before.

**Comment by jay0lee** [comment]
Fixed in 1.1

https://git.io/gyb-releases
